src/ModelManager.ts

```diff
@@ -180,6 +188,33 @@ export class ModelManager {
         return this._initPromise;
     }
 
+    /**
+     * Appends client lib to the page
+     */
+    private appendClientLibs(clientLibUrl : string) {
```

Comment:
Similarly to this function, stylesheet to support placeholders should also be added via `<link>`

Reply:
So basically we need to include css in a similar way?

Reply:
Added function to add stylesheets.